compiler/src/main/scala/org/scalajs/core/compiler/PrepJSExports.scala

```diff
@@ -726,7 +552,6 @@ trait PrepJSExports { this: PrepJSInterop =>
   /** Whether a symbol is an annotation that goes directly on a member */
   private lazy val isDirectMemberAnnot = Set[Symbol](
```

Comment:
Not necessarily. The "direct" here comes from "directly on the member" as opposed to "on the class".